scripts/jenkins-win64.bat

```diff
@@ -98,4 +98,9 @@ copy "%AIP%-SetupFiles\%AIP%.msi" "%targetsDir%\%archiveName%.msi"
 
 echo Building new installer with wix#
 call "BuildWixSharpInstaller.bat"
+
+if EXIST c:\electron-installer\signing.bat (
+	call c:\electron-installer\signing.bat
+)
+
```

Comment:
If it does not exist I think you should fail the build.
As it is now, we will just continue and release an unsigned build

Reply:
Good point. But the check was added so that you will be able to run the build script, including making the installer, on your local machine, where there is no certificate. But I guess ensuring that he build is always signed is more important?

Reply:
I agree with @mattias-symphony here. When we build on Jenkins, we know we'll have a signed build, so, there's a guarantee there. In any case, we don't distribute builds building locally but, it'll be useful for testing.

Reply:
Team decide we will exit if the signing tool is missing.

For locally run you will have the installer, but you will exit != 0

Reply:
Changed the behaviour as agreed